source/appModules/explorer.py

```diff
@@ -174,6 +174,8 @@ class UIProperty(UIA):
 	
 	def _get_value(self):
 		value = super(UIProperty, self).value
+		if value is None:

```

Comment:
Are you sure whether this is always None, i.e. why wouldn't this be an empty string?

Reply:
A never mind, I assume you got a "none type object has no attribute replace" error before?

Reply:
See https://github.com/nvaccess/nvda/pull/8316#issuecomment-390730470

Reply:
If you mean before applying my fix yes.